monitor/avs/avs.go

```diff
@@ -39,7 +39,7 @@ func Monitor(ctx context.Context, network netconf.Network) error {
 		return err
 	}
 
-	go monitorAVSOperatorsForever(ctx, avs)
+	startMonitoring(ctx, avs)
```

Comment:
I prefer `go startMonitoring(ctx, avs)` - no goroutines inside the method. This makes unit testing much easier.

Reply:
how would that work? each of the `monitorForever` needs to be a goroutine
Or we could just get rid of `startMonitoring` and move them all here

Reply:
I just saw that you do the same thing but for different bindings in parallel. 👍

Reply:
See suggestion below